app/src/main/java/org/eyeseetea/malariacare/layout/adapters/survey/FeedbackAdapter.java

```diff
@@ -202,6 +202,8 @@ private View getViewByCompositeScoreFeedback(CompositeScoreFeedback feedback, Vi
                 color = R.color.medium_score_color;
             }else if(scoreType.getClassification() == ScoreType.Classification.HIGH) {
                 color = R.color.high_score_color;
+            } else if (scoreType.getClassification() == ScoreType.Classification.NO_RESPONSE) {
+                color = R.color.textFieldColor;
```

Comment:
Exists a color palette for score, please create a new color in as no_score_color and its values be textfieldcolor